app/serializers/score_serializer.rb

```diff
@@ -1,3 +1,3 @@
 class ScoreSerializer < ActiveModel::Serializer
-  attributes :id, :username, :text_id, :wpm, :time, :accuracy, :created_at, :updated_at
+  attributes :id, :username, :text_id, :wpm, :time, :accuracy, :position, :created_at, :updated_at
```

Comment:
the problem with this is that when you make a new entry there all the positions get updated and that is not very efficient! That is why I made it the other way around so that I could see when the user submitted his result the highscore for that.